screens/Reader.js

```diff
@@ -94,18 +95,19 @@ class Reader extends React.Component {
     if (startBaniList.length > 0) {
       for (const bani of startBaniList) {
         if (bani.id == this.currentBani.id) {
-          if (bani.progress > 0 && bani.progress < 1) {
             progress = bani.progress;
-          }
         }
       }
     }
+    if(Number(progress)==1 || Number(progress)>"1"){
+      progress=0
+    }
```

Comment:
You are comparing the string with number. Convert the types to same type and use strong equality check.

Reply:
Fixed veer ji